index.js

```diff
@@ -26,14 +26,16 @@ async function resolveVersion(version, mirror) {
   let mirror = core.getInput("node-mirror") || "https://nodejs.org/dist/";
   let version = await resolveVersion(core.getInput("node-version"), mirror);
   if (process.platform == "win32") {
-    runScript("powershell", ".\\install.ps1", version, mirror);
+    let arch = core.getInput("node-arch") || null;
+    runScript("powershell", ".\\install.ps1", version, mirror, arch);
```

Comment:
Won't this pass the string "null" on the command line? Perhaps just ""?

Reply:
So true, I just updated it.